.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: "CI"
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "0 7 * * *"
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.OS }}-latest
+    name: "tests"
+    strategy:
+      matrix:
+        PYTHON:
+          - 3.8
+          - 3.9
+        OS:
+          - ubuntu
+          - windows
+          - macos
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        auto-update-conda: true
+        python-version: ${{ matrix.PYTHON }}
+        miniforge-variant: Mambaforge
+      - name: "Temporary installation (reqs and code)"
+        run: python -m pip install .[test]
+        # the above step should be changed to only install reqs
+      - name: "Versions"
+        run: conda list
+      - name: "Install"
+        run: python -m pip install --no-deps .
+      - name: "Check imports"
+        run: |
+          python -c "import openfe.setup"
```

Comment:
Should take this opportunity to switch this to a test like this so we can have our first pytest call (& get coverage, etc.. started)?

```python
from openfe import setup, orchestration, simulation, analysis
import sys

@pytest.mark.parametrize('module',
    ['setup', 'orchestration', 'simulation', 'analysis])
def test_imported(module):
    """Sample test, will always pass so long as import statement worked"""
    assert module in sys.modules
```